wamr-compiler/main.c

```diff
@@ -193,6 +193,11 @@ print_help()
     printf("Examples: wamrc -o test.aot test.wasm\n");
     printf("          wamrc --target=i386 -o test.aot test.wasm\n");
     printf("          wamrc --target=i386 --format=object -o test.o test.wasm\n");
+    printf("Usage for print architecture-related help infos: wamrc [--target/--cpu/--target-abi]=help\n");
+    printf("                                                 wamrc [--cpu-features]=+help\n");
```

Comment:
No need show help again here, `--target=help`/`--target-abi=help`/.. have been mentioned in each option:
```
Use --target=help to list supported targets
Use --target-abi=help to list all the ABI supported
Use --cpu=help to list all the CPU supported
Use --cpu-features=+help to list all the features supported 
```